components/MarkdownTextBlock/samples/MarkdownTextBlockCustomSample.xaml

```diff
@@ -15,6 +15,7 @@
         </Grid.ColumnDefinitions>
         <controls:MarkdownTextBlock
             Grid.Column="0"
+            Text="{x:Bind Text, Mode=OneWay}"
```

Comment:
Could just bind to `MarkdownTextBox.Text` directly? It's a bit weird for a UI class to implement `INotifyPropertyChanged`, and there's no real reason to do so here, so could just simplify the sample with a direct binding. The config isn't modified so it should be fine to have as the default, eh?

Reply:
Addressed in latet version